deployer/troubleshooting-integration/src/main/java/io/hyscale/troubleshooting/integration/conditions/IsApplicationCrashing.java

```diff
@@ -47,7 +47,7 @@ public boolean decide(TroubleshootingContext context) throws HyscaleException {
         }
 
         if (lastState != null) {
-            return lastState.equals(PodStatus.OOMKILLED.getStatus()) || lastState.equals(PodStatus.COMPLETED.getStatus());
+            return lastState.equals(PodStatus.OOMKILLED.getStatus()) || lastState.equals(PodStatus.COMPLETED.getStatus()) || lastState.equals(PodStatus.ERROR.getStatus());
```

Comment:
Would it be better to extract these conditions in a set and check if the last state exists in that set, in that case, we can also do away with a null check for the last state?